pandas/core/strings.py

```diff
@@ -481,11 +482,19 @@ def f(x):
             return [np.nan if item is None else item for item in m.groups()]
         else:
             return empty_row
+
     if regex.groups == 1:
-        result = Series([f(val)[0] for val in arr],
-                        name=_get_single_group_name(regex),
-                        index=arr.index, dtype=object)
+        if isinstance(arr, Index):
+            result = Index([f(val)[0] for val in arr],
+                           name=_get_single_group_name(regex),
+                           dtype=object)
+        else:
```

Comment:
I think its worthile to change ``_wrap_result`` to something like this:

```
def _wrap_result(self, result, **kwargs):
     if not isinstance(result, (Series, Index)):
         result = type(self.series)(result, **kwargs)
     .....
```

should work if ``expand=False``